### DISCUSSION TITLE
Simply capture a movie before and after motion

### DISCUSSION THREAD

**Discussion by saharco540** [Motion-Project/motionplus#93] [Q&A] [answered]
Hey all!
What would be the most basic, straight-forward configuration parameters so that whenever a motion is detected (putting aside noise configurations) a movie that is starts 1 second BEFORE to motion and X seconds AFTER the motion to be saved?
I am running a Pi Camera on an old Raspberry Pi.
Thanks!

**Comment by Mr-Dave** [comment, marked as answer]
You would have to determine that number yourself based upon your configuration, computer, camera(s) and resolutions.  Motion and Motionplus configuration parameters for pre and post capture require the number of frames.  As a result, you'll multiply the actual FPS that you are getting times number of seconds desired.  The number will NOT positively be the exact seconds since the actual FPS could vary depending upon many factors.
